mongodbatlas/service_version.go

```diff
@@ -0,0 +1,46 @@
+// Copyright 2021 MongoDB Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package mongodbatlas
+
+import (
+	"context"
+	"net/http"
+)
+
+const versionPath = "api/private/unauth/version"
+
+// ServiceVersionService is an interface for the version private endpoint of the MongoDB Atlas API.
```

Comment:
we tend to leave a note about this not being supported outside what can be seen here

https://github.com/mongodb/go-client-mongodb-atlas/blob/c271f9850cb0408445a31f2df0a920687c775524/mongodbatlas/ip_info.go#L26-L27